backend/routes/group.py

```
@@ -130,4 +130,14 @@ def get_posts_by_hashtag(db: Session = Depends(get_db)):
     # Return the sorted list of post ids
     return sorted_post_ids
 
+@router.get("/top-universities")
+def get_top_universities(limit: int = 5, db: Session = Depends(get_db)):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_top_universities`

(ANN201)